### DISCUSSION TITLE
Git LFS over limit, but repo has no files

### DISCUSSION THREAD

**Discussion by ph34rl3ssL34d3r** [community/community#45747] [Repositories] [answered]
### Select Topic Area

Question

### Body

Greetings,

I've been trying to set up a repository for a university game project with 5 members.
One member zipped up around 3GB of Unity files and sent to me after they couldn't upload.

After all attempts to push have been silently rejected, an email saying I'd gone over my LFS file limit was finally received.
Currently the repository called Helpful_Poltergeist shows it should have no files.
My account says I'm using 1.42 of 1 GB of storage.
What can I do to clear it out and try again with a leaner unity project file size?

Thank you,

Brent

edit: repository deletion not permitted
![nodelete](https://user-images.githubusercontent.com/13506865/215386131-cb8332ce-5f4f-4f41-bf38-770d42edb185.jpg)

**Comment by jge162** [comment, marked as answer]
Did you copy paste in the repo name? If you did it might have a white space in front of p

**Reply by ph34rl3ssL34d3r** [reply]
I typed it in manually using a Google chrome and MS edge browsers.
Neither allowed me to continue further.

**Reply by jge162** [reply]
> I typed it in manually using a Google chrome and MS edge browsers.
> Neither allowed me to continue further.

![C481BE8B-1DEC-4A26-A30C-C1E2DD1A019C](https://user-images.githubusercontent.com/31228460/215408903-5d8e60f8-5641-49b8-a131-a824750dc2d8.png)

you are misspelling ‘poler’ with ‘poltergeist’

**Reply by jge162** [reply]
Typos @ph34rl3ssL34d3r

**Reply by ph34rl3ssL34d3r** [reply]
Helps to have another set of eyes, thank you!

**Reply by jge162** [reply]
Were you able to delete?

**Reply by jge162** [reply]
If you no longer need help you can select answered to close ticket, if you like. Thank you again.
